ajax/libs/jquery.wookmark/package.json

```diff
@@ -12,5 +12,14 @@
     "url": "https://github.com/germanysbestkeptsecret/Wookmark-jQuery"
   },
   "license": "MIT",
-  "author": "Christoph Ono <chri@sto.ph>"
+  "author": "Christoph Ono <chri@sto.ph>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/germanysbestkeptsecret/Wookmark-jQuery.git",
+    "basePath": "",
+    "files": [
+      "wookmark.min.js",
```

Comment:
This lib also has npm package, would you please use npm auto-update config? It's more efficient.